src/main/webapp/chat.html

```diff
@@ -2,11 +2,20 @@
 <head>
   <title>Pusher Test</title>
   <script src="https://js.pusher.com/6.0/pusher.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.js"></script>
   <script src="chat.js"></script>
+  <link rel="stylesheet" type="text/css" href="chat.css" />
```

Comment:
Can you rename the stylesheet styles.css, since we hope to use one across pages?